airbyte-config/config-persistence/src/test/java/io/airbyte/config/persistence/ConfigRepositoryTest.java

```diff
@@ -1,511 +0,0 @@
-/*
```

Comment:
all tests in this class were moved into separate `*PersistenceTest` (`HealthCheckPersistenceTest.java` `WorkspacePersistenceTest.java`, and `ActorDefinitionPersistenceTest.java`)